approval_polls/templates/approval_polls/edit.html

```diff
@@ -16,111 +16,113 @@
-              </div>
-              <div class='checkbox'>
-                <label><input type='checkbox' name='show-countdown' id='checkbox2' value='' autocomplete='off' {% if poll.show_countdown%}checked{% endif %}>Display Poll Closing Countdown with this poll.</label>
-              </div>
-<!--
-              <hr>
-              <p>Customize Poll</p>
-              <div class='checkbox'>
-                <label><input type='checkbox' name='show-write-in' value=''>Permit Write-in option with this poll.</label>
-              </div>
-              <div class='checkbox'>
-                <label><input type='checkbox' name='show-lead-color' value='' checked>Show Leading option in different color.</label>
-              </div>
+                <p>Poll Visibility</p>
+                <div class='checkbox'>
+                  <label><input type='checkbox' id='poll-visibility' name='public-poll-visibility' value='' autocomplete='off' {% if poll.is_private %}{% else %}checked{% endif %}>List this poll on the home page.</label>
+                </div>
+                <hr>
+                <div class = 'datetimepicker'>
+                  <p>Please select a closing date and time for this poll, if any</p>
+                  <input id='datetimepicker' type='text' class='form-control' name='close-datetime' value='{{closedatetime}}' placeholder='Poll Closing Date' style='background-color: white;' readonly>
+                </div>
+                <div class='checkbox'>
+                  <label><input type='checkbox' name='show-close-date' id='checkbox1' value='' autocomplete='off' {% if poll.show_close_date%}checked{% endif %}>Display Poll Closing Date with this poll.</label>
+                </div>
+                <div class='checkbox'>
+                  <label><input type='checkbox' name='show-countdown' id='checkbox2' value='' autocomplete='off' {% if poll.show_countdown%}checked{% endif %}>Display Poll Closing Countdown with this poll.</label>
+                </div>
+<!--  
+                <hr>
+                <p>Customize Poll</p>
+                <div class='checkbox'>
+                  <label><input type='checkbox' name='show-write-in' value=''>Permit Write-in option with this poll.</label>
+                </div>
+                <div class='checkbox'>
+                  <label><input type='checkbox' name='show-lead-color' value='' checked>Show Leading option in different color.</label>
+                </div>
 -->
+              </div>
             </div>
           </div>
         </div>
-      </div>
-      <button type='submit' class='btn btn-success pull-right'>Save</button>
-    </form>
+        <button type='submit' class='btn btn-success pull-right'>Save</button>
+      </form>
+    {% else %}
+      <h3>The poll cannot be edited as it has ballots on it!</h3>
```

Comment:
With this change the user is not able to edit anything about the poll even if it has a single ballot on it. I think we should allow the poll owner to change the other parameters such as poll close date and visibility on the home page. However we would not want to allow the text on the choice or poll question to be edited.